test/unit/contract_test.js

```diff
@@ -7,21 +9,6 @@ describe('Contract', function () {
       expect(contract.contractId('strkey')).to.equal(contractId);
     });
 
-    it('parses hex addresses', function () {
```

Comment:
let's make this `throws on deprecated hex addresses`, instead?

Reply:
Sure. I'll make it say "obsolete", since "deprecated" means "still technically supported, but will be removed soon."

Reply:
See below